2019-Q4-PyPI/RFP.md

```diff
@@ -0,0 +1,197 @@
+  - Freelance or firm? Incorporation? Subcontracting?
+  - Free/Open Source Software experience?
+
+- Agreement to [project management and reporting](#project-management-and-reporting) requirements.
+
+- What Milestone(s) are you proposing for?
+  - We recommend proposing for all of the work in scope of Milestone 1, Milestone 2, or both.
+
+- Examples of similarly-complex projects completed previously.
+  - Referencing contributions to Free/Open Source projects is encouraged.
+
+- Project timeline estimates by milestone and task. These timelines should fit within our [project timeline](#project-timeline), however proposers may suggest adjustments based on their availability.
+  - Milestone 1 - Verifiable cryptographic signing of artifacts
+  - Milestone 2 - Systems for Automated Detection of Malicious Uploads
+
+- Project budget by milestone and task. Deviations from our [estimated budgets and caps](#estimated-budgets-and-caps) should be described and supported.
+  - Milestone 1 - Verifiable cryptographic signing of artifacts
+  - Milestone 2 - Systems for Automated Detection of Malicious Uploads
+
+## Evaluation criteria
+
+### Potential pass/fail:
+
+ - Contains all elements specified in the [elements of the proposal](#elements-of-proposal).
+ - Proposal is detailed enough to properly assess further criteria.
+ - Formatting and Submission requirements:
+   - Portable Document Format (PDF)
+   - Emailed to [ernest@python.org](mailto:ernest@python.org) by **October 16, 2019 [AoE](https://www.timeanddate.com/time/zones/aoe)** (2019-10-16T12:00:00Z)
+
+### Experience and competency
+
+- Does the proposal demonstrate relevant experience necessary to complete the work?
+- Is there demonstrable experience with enough of the [relevant technologies](#specific-technologies-used) for each Milestone to support timelines?
+- Do the examples of similarly complex projects and any references to past Free/Open Source Software contributions indicate competency?
+
+### Cost
+
+- Does the budget proposed fit within our [estimated budgets and caps](#estimated-budgets-and-caps)?
+- Are the milestone and task budgets reasonable and competitive?
+- Are any deviations from the [estimated budgets and caps](#estimated-budgets-and-caps) well supported and explained?
+
+### Process and timeline
+
+- Do the proposers agree to the [project management and reporting](#project-management-and-reporting) requirements?
+- Are the project timeline estimates within our [project timeline](#project-timeline)? Are deviations from our timeline explained and supported?
+
+## Scope
+
+This Request for Proposals is seeking **backend developers** to implement, test, verify, and assist in the rollout of the following features to the codebase that powers [PyPI](https://pypi.org/).
+
+Discussions leading to these milestones can be read in the [forum for our Request for Information](https://discuss.python.org/c/python-software-foundation/pypi-q4-rfi).
+
```

Comment:
I'd like to add a note here about what is NOT in scope:

"Not in scope: adding TUF metadata generation or reading functionality elsewhere in the packaging toolchain (setuptools, pip, etc.) or surfacing malware detection results to PyPI packagers and downloading users via the web UI or API"